cases/templates/cases/case_detail_staff.html

```diff
@@ -75,6 +75,23 @@ <h2 class="lbh-heading-h2 section-heading">Case info</h2>
       </a>
     </dd>
   </div>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Followers</dt>
+    <dd class="govuk-summary-list__value">
+      {% for follower in case.followers.all %}
+        {% if forloop.first %}<ul class="govuk-list govuk-list--bullet">{% endif %}
+        <li><a href="{% url "cases" %}?assigned={{ follower.id }}" class="nw-link--no-visited-state">{{ follower }}</a>
+        {% if forloop.last %}</ul>{% endif %}
+      {% empty %}
+        None
+      {% endfor %}
+    </dd>
+    <dd class="govuk-summary-list__actions">
+    <a class="nw-link--no-visited-state" href="{% url "case-followers" case.id %}">
```

Comment:
I can see why we might want to allow super admins or something to modify who’s following a case, but my instinct is most users will just want a "follow" / "unfollow" button here, to manage _their own_ following status, rather than being taken to a screen where they have to find themselves somewhere in the list of 40+ users (and also, weirdly, where they’re able to forcibly _unfollow_ someone else from a case that person presumably care about).

Reply:
Yeah, that's probably a better thing to begin with, isn't it.

Reply:
Have switched to have a button like that (form is still there for potential future super admin use).